daemon/algod/api/algod.oas2.json

```diff
@@ -468,6 +468,61 @@
         }
       }
     },
+    "/v2/transactions/dryrun": {
+      "post": {
+        "consumes": [
+          "application/x-msgpack",
+	  "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "schemes": [
+          "http"
+        ],
+        "summary": "Provide debugging information for a transaction (or group).",
+        "operationId": "TransactionDryRun",
+        "parameters": [
```

Comment:
we've also discussed to take programs source and the protocol spec.
If a program can be set outside of the txn then also group-index parameter is needed.

Reply:
Added a way to upload source that gets inserted into LogicSig or App Params